CIP-????/README.md

```diff
@@ -13,101 +13,101 @@ License: CC-BY-4.0
 ---
 
 ## Abstract
-In the absence of atomic delegation, in order for layer 1 (L1) Dapp users to maintain full delegation control of their assets, each user must have his/her own address while using the Dapp. This poses a challenge: if all users have their own addresses while using the Dapp, how can users find and interact with each other, without relying on a central order batcher? This informational CIP proposes an NFT standard, called ***Beacon Tokens***, to solve this broadcasting issue. Using beacon tokens, it is possible to create distributed L1 Dapps (eg, DEXs, p2p lending, etc) where users not only maintain full delegation control of their assets, but can also maintain full custody of their assets at all times. Beacon tokens can be generalized to other use cases, such as creating an on-chain personal address book tied to - and fully recoverable by - a user's payment pubkey, or trustlessly sharing reference scripts with other blockchain users.
+This CIP outlines a native token standard, called Beacon Tokens, that serves as a way to "tag" on-chain data in a way that is efficiently queryable by off-chain APIs. This enables the creation of **massively** concurrent dApps that require contextual availability of on-chain data, such as programmable P2P swaps. 
```

Comment:
I'm not sure **massively** is appropriate, especially given the redundant execution problem. I think **organically** would be better.